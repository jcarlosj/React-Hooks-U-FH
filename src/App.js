import React from 'react';

export const App = () => {
    return (
        <h1>Hooks App</h1>
    )
}
