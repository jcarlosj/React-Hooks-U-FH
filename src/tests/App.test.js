import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

import { App } from '../App';

describe( '<App />', () => {

    test( 'debe desplegar correctamente', () => {
        
        const wrapper = shallow( 
            <App /> 
        );
        expect( wrapper ).toMatchSnapshot();

    });

    test( 'debe desplegar titulo "Hooks App"', () => {
        
        const 
            wrapper = shallow( 
                <App /> 
            ),
            titleText = wrapper.find( 'h1' ).text();

        // console.log( titleText );
        expect( titleText ).toBe( 'Hooks App' );

    });

} );