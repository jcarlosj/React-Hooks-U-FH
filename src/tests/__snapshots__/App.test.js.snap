// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> debe desplegar correctamente 1`] = `
<h1>
  Hooks App
</h1>
`;
